integration-cli/docker_cli_netmode_test.go

```diff
@@ -14,152 +12,80 @@ import (
 // the command executed in a container did really run PS correctly.
 const stringCheckPS = "PID   USER"
 
-// checkContains is a helper function that validates a command output did
-// contain what was expected.
-func checkContains(expected string, out string, c *check.C) {
-	if !strings.Contains(out, expected) {
-		c.Fatalf("Expected '%s', got '%s'", expected, out)
-	}
+// DockerCmdWithFail executes a docker command that is supposed to fail and returns
+// the output, the exit code. If the command returns an Nil error, it will fail and
+// stop the tests.
+func dockerCmdWithFail(c *check.C, args ...string) (string, int) {
```

Comment:
@runcom maybe but I think this is not the right PR to do it. I'm gonna work on this (with starting integration test with go & co) but I think we shouldn't have not too much `dockerCmdWith..` function :stuck_out_tongue: